src/Support/Arr/Blacklist.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * This file is part of the Phalcon.
+ *
+ * (c) Phalcon Team <team@phalcon.com>
```

Comment:
```suggestion
 * (c) Phalcon Team <team@phalcon.io>
```

Reply:
Let me do a global replace in case I missed this elsewhere

Reply:
Resolved with latest push